app/components/pages/SubmissionWizard/steps/Files/ManuscriptUpload.js

```diff
@@ -135,34 +133,67 @@ const DropzoneContent = ({ conversion, formError, dropzoneOpen }) => {
   )
 }
 
-const FileUpload = ({ onDrop, conversion, formError, ...props }) => {
-  let dropzoneRef
-  return (
-    <StyledDropzone
-      accept={VALID_FILE_TYPES}
-      onDrop={onDrop}
-      {...props}
-      disableClick
-      hasError={!!(formError || conversion.error)}
-      maxSize={config.fileUpload.maxSizeMB * 1e6}
-      saveInnerRef={node => {
-        dropzoneRef = node
-      }}
-    >
-      <CentredFlex>
-        <Box width={1}>
-          <DropzoneContent
-            conversion={conversion}
-            dropzoneOpen={() => dropzoneRef.open()}
-            formError={formError}
-          />
-        </Box>
-      </CentredFlex>
-    </StyledDropzone>
-  )
+class ManuscriptUpload extends React.Component {
+  state = {
+    errorMessage: null,
+  }
+
+  componentDidUpdate(prevProps) {
```

Comment:
Would be good to have tests that cover all the scenarios in here, since this is something new that's being added with this PR